src/node.rs

```diff
@@ -289,6 +332,33 @@ impl<S: StorageExt> SawtoothRaftNode<S> {
                 if entry.get_entry_type() == EntryType::EntryNormal {
                     let block_id: BlockId = BlockId::from(Vec::from(entry.get_data()));
                     self.block_queue.add_block_commit(block_id);
+                } else if entry.get_entry_type() == EntryType::EntryConfChange && entry.get_term() != 1 {
+                    let change: ConfChange = protobuf::parse_from_bytes(entry.get_data())
+                        .expect("Failed to parse ConfChange");
+                    let raft_id = change.get_node_id();
+                    info!("Configuration change received: {:?}", change);
+
+                    // Update raft_id_to_peer_id according to change
+                    match change.get_change_type() {
+                        ConfChangeType::RemoveNode => {
+                            self.raft_id_to_peer_id.remove(&raft_id).unwrap();
+                        },
+                        ConfChangeType::AddNode => {
+                            self.raft_id_to_peer_id.insert(
+                                raft_id,
+                                PeerId::from(Vec::from(change.get_context()))
+                            );
+                        },
+                        _ => {}
+                    }
+
+                    self.raw_node.apply_conf_change(&change);
```

Comment:
I would suggest moving L338 to L355 behind a private method, something like `apply_conf_change(change: &ConfChange)` is probably good.